docs/paper/dev/misc/databases.md

```diff
@@ -0,0 +1,107 @@
+---
+slug: dev/using-databases
+---
+
+# Using Databases
+
+When you are storing larger amounts of data inside a plugin, we recommend using a database. This guide will walk you through the startup process.
+
+## MySQL vs SQLite
+
+There are two main types of database software that you could use: MySQL vs SQLite. They both have many pros and cons outlined [here](https://www.hostinger.com/tutorials/sqlite-vs-mysql-whats-the-difference/)
+in more depth, however the main difference is that MySQL requires external database software running, whereas SQLite is the entire package. Both are explained below.
+
+## SQLite
+
+To work with SQLite you will need a driver to connect / initialise the database. 
+We will be recommending the JDBC connector which can be added to your project with the following dependency:
+
+### Maven:
+```xml
+<dependency>
+    <groupId>org.xerial</groupId>
+    <artifactId>sqlite-jdbc</artifactId>
+    <version>3.40.0.0</version>
+    <scope>provided</scope>
+</dependency>
+```
+
+### Kotlin Gradle:
+```kotlin
+dependencies {
+    compileOnly("org.xerial:sqlite-jdbc:3.40.0.0")
+}
+```
+
+:::info
+
+The JDBC Driver is bundled with Paper, so you do not need to shade/relocate it.
+
+:::
+
+### Usage
+
+From here, you must invoke a `Class.forName` on the driver to allow it to initialise and then create the connection to the database:
+
+```java
+public class DatabaseManager {
+    
+    public void connect() {
+        Class.forName("org.sqlite.JDBC");
+        Connection connection = DriverManager.getConnection("jdbc:sqlite:plugins/TestPlugin/database.db");
+    }
+}
+```
+
+To learn more about the Java Database Connection, see [here](https://www.w3schools.blog/jdbc-tutorial)
```

Comment:
w3schools is a really meh resource (and that link isnt even the real one?)
I would suggest this as a quick primer https://www.baeldung.com/java-jdbc and https://docs.oracle.com/javase/tutorial/jdbc/basics/index.html for a fully featured reference